docs/Project.toml

```diff
@@ -3,12 +3,16 @@ Augmentor = "02898b10-1f73-11ea-317c-6393d7073e15"
 DemoCards = "311a05b2-6137-4a5a-b473-18580a3d38b5"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
+ISICArchive = "0f918735-7648-5aeb-9b51-2c108d137345"
```

Comment:
I guess this is not used anymore?

```suggestion
```

Reply:
It is still used in Getting started. I think we can keep it, it looks better when there are more kinds of images, not just MNIST.